build.sbt

```diff
@@ -62,17 +60,17 @@ val `faithful-cats` =
     .settings(
       crossScalaVersions := crossScalaV,
       libraryDependencies ++= Seq(
-        "org.typelevel" %%% "cats-core" % "2.0.0",
-        "org.typelevel" %%% "cats-laws" % "2.0.0" % Test,
-        "org.typelevel" %%% "discipline-scalatest" % "1.0.0-M1" % Test,
-        "org.scalatest" %%% "scalatest" % "3.1.0-M2" % Test
+        "org.typelevel" %%% "cats-core" % "2.2.0-M2",
```

Comment:
Is there a non-milestone release that supports Scala.js 1.x?

Reply:
yes, use `2.1.1` the the new commit